pkg/monitortests/client/api_unreachable.go

```diff
@@ -0,0 +1,164 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	routeclient "github.com/openshift/client-go/route/clientset/versioned"
+	utilmetrics "github.com/openshift/library-go/test/library/metrics"
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/monitortestframework"
+	"github.com/openshift/origin/pkg/monitortests/metrics"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	prometheustypes "github.com/prometheus/common/model"
+)
+
+const (
+	MonitorName = "api-unreachable-from-client"
+)
+
+func NewAPIUnReachableFromClientMonitorTest() monitortestframework.MonitorTest {
```

Comment:
Godoc would be nice here. Initial questions I had looking over this, what client, where the metrics come from, how we interpret them to generate these intervals, and what those intervals mean.